server/routes/termdb.cluster.ts

```diff
@@ -1,5 +1,4 @@
 import path from 'path'
-import fs from 'fs'
 import run_R from '#src/run_R.js'
 import {
 	TermdbClusterRequest,
```

Comment:
please also modify TermdbClusterRequest to declare new parameters e.g. clusterMethod